benchmarks/haskell/Benchmarks/EncodeUtf8.hs

```diff
@@ -23,6 +25,8 @@ benchmark name string =
     bgroup name
         [ bench "Text"     $ whnf (B.length . T.encodeUtf8)   text
         , bench "LazyText" $ whnf (BL.length . TL.encodeUtf8) lazyText
+        , bench "Text/encodeUtf8Builder" $ whnf (B.toLazyByteString . T.encodeUtf8Builder) text
+        , bench "Text/encodeUtf8BuilderEscaped" $ whnf (B.toLazyByteString . T.encodeUtf8BuilderEscaped (BP.liftFixedToBounded BP.word8)) text
```

Comment:
This looks a bit weird: `whnf` of a lazy bytestring does not trigger anything except the first chunk.

Reply:
Ah, right!